src/index.js

```diff
@@ -0,0 +1,71 @@
+import './style.css';
+import './style.scss';
+import 'bootstrap';
+
+const toDolist = document.getElementById('toDoList');
+const addBtn = document.getElementById('addBtn');
+const inputs = document.getElementsByClassName('forminput');
+let listArray = [
+  {
+    description: 'AddList',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'RemoveList',
+    completed: true,
+    index: 2,
+  },
+  {
+    description: 'ClearAllList',
+    completed: false,
+    index: 3,
+  },
+  {
+    description: 'reunServer',
+    completed: true,
+    index: 4,
+  },
+];
+
+function addList() {
+  const listarr = {
+    name: toDolist.value,
+  };
+  listArray.push(listarr);
+}
+
+function displayData() {
+  let trs = '';
+  for (let i = 0; i < listArray.length; i += 1) {
+    trs += `<tr>
+     <td><input type="checkbox"></td>
+    <td class="float-left">${listArray[i].name}</td>
+    <td><button onclick="deleteList(${i})" class="float-right mt-lg-1"><i class="fa fa-ellipsis-v"></i></button></td>
+    </tr>`;
+  }
+  document.getElementById('tableBody').innerHTML = trs;
```

Comment:
- [ ] Please make sure you render properly your array items to the front end, actually, it returns `unddefined`
<img width="901" alt="arry" src="https://user-images.githubusercontent.com/79058364/160146084-db7d13bb-070c-4e69-aec8-6774e96f01cc.png">